libclamav/ole2_extract.c

```diff
@@ -1306,7 +1306,7 @@ static cl_error_t handler_enum(ole2_header_t *hdr, property_t *prop, const char
                     } else {
                         if (!ole2_read_block(hdr, hwp_check, 1 << hdr->log2_big_block_size, prop->start_block)) {
                             status = CL_EREAD;
-                            break;
+                            goto done;
```

Comment:
I don't think we want to do this. I think the correct solution is to remove status = CL_EREAD; instead.